src/.vuepress/config.js

```diff
@@ -1,14 +1,14 @@
 const sidebar = {
   cookbook: [
     {
-      title: 'Cookbook',
+      title: 'Recetario',
```

Comment:
I think we should use `Libro de Recetas` to keep it consistent with the current docs
```suggestion
      title: 'Libro de Recetas',
```